locales/en/app.ftl

```diff
@@ -259,3 +261,182 @@ share-twitter = Most people have about 100 online accounts. Have any of yours be
+
+user-scan-results-headline =
+  { $breachCount ->
+        [one] { $userEmail } appeared in 1 data breach.
+       *[other] { $userEmail } appeared in { $breachCount } data breaches.
+  }
+
+guest-scan-results-headline =
+  { $breachCount ->
+        [one] This email appeared in 1 data breach.
+       *[other] This email appeared in { $breachCount } data breaches.
+  }
+
+user-no-breaches-blurb = We’ll alert you if this email address appears in a new breach.
+
+guest-no-breaches-blurb = 
+  To see if this email appears in sensitive breaches, create a {-brand-fxa}. 
+  We’ll also alert you if this address appears in new data breaches.
+
+user-one-breach-blurb = This breach exposed the following personal info.
+
+user-fb-compromised-blurb =
+  { $breachCount ->
+      [one] Your email also appeared in { $breachCount } other breach.
+     *[other] Your email also appeared in { $breachCount } other breaches.
+  }
+
+user-generic-fb-compromised-blurb =
+  { $breachCount ->
+      [one] This email also appeared in { $breachCount } other breach. 
+     *[other] This email also appeared in { $breachCount } other breaches.
+  }
+
+user-fb-compromised-single = 
+  This breach exposed the following personal info. If you haven't already, 
+  change your passwords.
+user-generic-fb-compromised-single = This breach exposed the following personal info.
+guest-fb-compromised-single = 
+  This breach exposed the following personal info. Sign up for a {-brand-fxa} 
+  to get your full report and alerts about new breaches.
+
+guest-fb-compromised-blurb =
+  { $breachCount ->
+      [one] This email also appeared in { $breachCount } other breach. Sign up for a 
+            {-brand-fxa} to get your full report and alerts about new breaches.
+     *[other] This email also appeared in { $breachCount } other breaches. Sign up for a 
+              {-brand-fxa} to get your full report and alerts about new breaches.
+  }
+
+user-fb-not-compromised-blurb =
+  { $breachCount ->
+      [one] You were not in the { $breachName } breach, but we found that email address in another.
+     *[other] You were not in the { $breachName } breach, but we found that email address in others.
+  }
+
+user-generic-fb-not-compromised-blurb =
+  { $breachCount ->
+      [one] This email was not in the { $breachName } breach, but we found this email address in another.
+     *[other] This email was not in the { $breachName} breach, but we found this email address in others.
+  }
+
+guest-fb-not-compromised-blurb =
+  { $breachCount ->
+        [one] This email was not in the { $breachName } breach, but we found that email address in another. 
+              Sign up for a {-brand-fxa} to get your full report and alerts about new breaches.
+       *[other] This email was not in the { $breachName } breach, but we found that email address in others. 
+                Sign up for a {-brand-fxa} to get your full report and alerts about new breaches.
+  }
+
+user-found-breaches-blurb =
```

Comment:
Since en-US doesn't use the `$breachCount` variable, can you add a note to these 3 strings to explain that locales can add it as needed. E.g.

```
# While English doesn’t use the actual number of breaches in this sentence,
# you can use {$breachCount} to display the number of breaches in your localization.
```